.github/workflows/ci.yml

```diff
@@ -125,7 +125,7 @@ jobs:
             echo "$CURRENT_DIR/$COMPONENT"
             cd "$CURRENT_DIR/$COMPONENT"
 
-            sed -i -re 's/"require": \{/"minimum-stability": "dev","prefer-stable": true,"require": \{/' composer.json
+            sed -i -re 's/"require": \{/"minimum-stability": "alpha","prefer-stable": true,"require": \{/' composer.json
```

Comment:
I don't get why prefer-lowest failed without this change on downloading `2.0.0-alpha.2`

Reply:
Hm.. Neither do I.. 

But this change will fail when S3 is require a version of core that is not released yet. (When it is only present in branch alias)